### PR TITLE
vpn-slice: update to 0.16.1.

### PR DESCRIPTION
**Pull request by maybe-one-day-ubermensch**
#### Testing the changes
- I tested the changes in this PR: **YES**

### CODE REVIEW

File: srcpkgs/vpn-slice/template [hunk 1]
```diff
@@ -1,6 +1,6 @@
 # Template file for 'vpn-slice'
 pkgname=vpn-slice
-version=0.15
+version=0.16.1
 revision=2
```
Comment (abenson) [inline srcpkgs/vpn-slice/template:4]: `revision` should be 1

**Comment by abenson** [comment]
After `makedepends`, add `checkdepends="$depends python3-pytest"`

**Comment by maybe-one-day-ubermensch** [comment]
There aren't any tests defined so I disabled the checks

File: srcpkgs/vpn-slice/template [hunk 2]
```diff
@@ -10,4 +10,6 @@ maintainer="Jony <maybe-one-day-ubermensch@protonmail.com>"
 license="GPL-3.0-or-later"
 homepage="https://github.com/dlenski/vpn-slice"
 distfiles="https://github.com/dlenski/${pkgname}/archive/v${version}.tar.gz"
-checksum=9e6f2fc9c1305d78e804529ddc7e787a88d185ba7a3560218ab5cdee4233a757
+checksum=ecdb1c434c611427447b4884ea46845584ab08e40687e3dd4ffbdb7cd9bbcbde
+# There aren't any tests defined
+make_check=no
```
Comment (ericonr) [inline srcpkgs/vpn-slice/template:15]:
```suggestion
make_check=no # there aren't any tests defined
```

That way it's easier to see why tests are disabled when grepping for `make_check=`.

Comment (maybe-one-day-ubermensch) [reply srcpkgs/vpn-slice/template]: Done!

**Pull request merged**